BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -94,7 +94,11 @@ void BSP_CAN1_Init(uint16_t* idWhitelist, uint8_t idWhitelistSize) {
     CAN_InitStruct.CAN_NART = DISABLE;
     CAN_InitStruct.CAN_RFLM = DISABLE;
     CAN_InitStruct.CAN_TXFP = ENABLE;
+    #ifdef LOOPBACK
```

Comment:
I think that it would be helpful to add a comment here too, especially after adding the MotorCAN and CarCAN values as Maddie and Ishan have suggested.